test/ngyn-server-connection-spec.js

```diff
@@ -280,21 +280,30 @@
       backend.completeConnection();
 
       ChatMessages.server.say( 'hello' );
-      expect( backend.server( 'ChatMessages' ).say.callCount ).toBe( 1 );
+      expect( backend.serverMethods.ChatMessages.say.callCount ).toBe( 1 );
     } ) );
     
     it( 'invocation promise should occur inside a $digest', inject( function( $rootScope) {
       var scope = $rootScope.$new();
-      var inDigest = false;
+      var digestOccurred, callbackExecuted = false;
       backend.addServerMethods( 'ChatMessages', { say: function() { angular.noop } } );
       
       ChatMessages.connect( scope );
+      
       backend.completeConnection();
       
-      ChatMessages.server.say( 'hello' ).then( function() { inDigest = !!scope.$$phase  } ) ;
+      ChatMessages.server.say( 'hello' ).then( function() { 
+        callbackExecuted = true;
+      } ) ;
+
+      scope.$watch( function() {
+        digestOccurred = true;
+      } );
+      
       backend.flush();
         
-      expect( inDigest ).toBe( true );
+      expect( callbackExecuted ).toBe( true );
+      expect( digestOccurred ).toBe( true );
     } ) );
   } );
```

Comment:
This no longer runs within an apply, but we do expect an apply to of been called. I check that by seeing if any watchers trigger